src/persistence/basin_subzone.js

```diff
@@ -159,5 +159,48 @@ module.exports = (
       )
         .then(layers => layers.rows[0].collection)
     ),
+
+    /**
+     * Get the current human footprint layer divided by categories in a given basin subzone
+     * @param {Number} subzoneId basin subzone id
+     * @param {Number} year optional year to filter data, 2018 by default
+     *
+     * @return {Object} Geojson object with the geometry
+     */
+    findHFCategoriesLayerById: (subzoneId, year = 2018) => (
+      db.raw(
+        `SELECT row_to_json(fc) AS collection
+        FROM (
+          SELECT 'FeatureCollection' AS type, array_to_json(array_agg(f)) AS features
+          FROM(
+          SELECT 
+            'Feature' AS TYPE,
+            row_to_json(prop) AS properties,
+            ST_AsGeoJSON(ST_SimplifyPreserveTopology(geom, ?))::json AS geometry 
+          FROM (
+            SELECT 
+              ST_Collect(geom) AS geom,
+              hf_cat AS key
+            FROM geo_hf
+            WHERE id_subzone = ?
+              AND hf_year = ?
+            GROUP BY key
+            ) AS geo
+            INNER JOIN (
+              SELECT 
+                hf_cat AS key,
+                sum(area_ha) AS value
```

Comment:
what do you think of naming the attribute "area" instead of "value"

Reply:
I agree. We must take into account that by changing the name of this attibute it is neccesary to update the function highlightFeature en Search component because there we call that attribute as "value" to render the tooltip on the map

Reply:
Hmm, good point. In order to keep that behavior uniform, we could let "value" as it is.

Reply:
Change is already done

Reply:
Oh, but that change in the front would imply to have special cases for this layers, so the correct would be to change them all... I think it's better this time to stick with value, no?

Reply:
Ah no!, don't worry. I see there are already special cases for all layers. So it's good to name it "value". I'll change that logic in my branch and the mockup :)

Reply:
In order to keep uniformity with the rest of endpoints that returns area values, i suggest let this attribute as "area" and update the front-end in the same issue when we conect all endpoints with the front. What do you think?

Reply:
yeap, I forgot the endpoints are not connected and nothing is going to break yet